tests/test.hs

```diff
@@ -158,7 +158,7 @@ knownToFail Z3   = [ "tests/pos/linspace.hs" ]
 testCmd :: FilePath -> FilePath -> FilePath -> SmtSolver -> LiquidOpts -> String
 ---------------------------------------------------------------------------
 testCmd bin dir file smt (LO opts)
-  = printf "cd %s && %s --verbose --smtsolver %s %s %s" dir bin (show smt) file opts
+  = printf "cd %s && %s --smtsolver %s %s %s" dir bin (show smt) file opts
```

Comment:
Fair enough, let's just add `--verbose` to the travis scripts then. It has been invaluable on the (rare) occasion that I can't reproduce the failure locally.